.kitchen.yml

```diff
@@ -19,12 +18,14 @@ driver:
 
 provisioner:
   name: chef_zero
+  config:
+    log_level: debug
```

Comment:
Do we want debug level here or was this meant for testing only?

Reply:
good catch ;-)